app/models/region.rb

```diff
@@ -156,9 +156,7 @@ def syncable_patients
     case region_type
       when "block"
         registered_patients.with_discarded
-          .select(:id, :registration_facility_id)
-          .or(assigned_patients.with_discarded.select(:id, :registration_facility_id))
-          .union(appointed_patients.with_discarded.select(:id, :registration_facility_id))
+          .union(assigned_patients.with_discarded)
```

Comment:
There a few notable changes here.
* We were selecting only a handful of columns before, and we no longer are
* `.or(assigned_patients...` is now `.union(assigned_patients...`

Can you talk me through why these changes are made, or aren't important?

Reply:
We were doing a `union` on a restricted set of columns earlier because it's expensive to union on all columns. We were then doing a subquery like so to get back all the columns
```
Patient.where(id: current_sync_region.syncable_patients)
```
It turns out it's faster to do a `union` on all columns (new) than doing the `union` + subquery lookup (old). The query is now doing 
```
current_sync_region.syncable_patients.updated_on_server_since ...
```

It's also faster than doing an `or`. (plan attached)

Subquery with union: https://explain.depesz.com/s/ebl0 (slow)
Or: https://explain.depesz.com/s/VLwB (slower)
Union: https://explain.depesz.com/s/BUYM (fastest)

Reply:
Thank you for this explanation, very helpful! Let's do this 😎